.github/workflows/release-create-hotfix.yml

```diff
@@ -15,17 +27,31 @@ jobs:
         with:
           app_id: ${{ secrets.GH_BOT_APP_ID }}
           private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v4
+        with:
+          ref: develop
+          sparse-checkout: tools/actions/composites
+
+      - name: Generate ref/tag version to use during main checkout
+        uses: ./tools/actions/composites/generate-tag
+        id: format-app-tag
+        with:
+          tag_version: ${{ github.event.inputs.tag_version }}
+          application: ${{ github.event.inputs.application }}
+        
       - uses: actions/checkout@v4
```

Comment:
you may need to specify fetch depth here too

Reply:
as above - fetch-depth defaults to 1 which is fine for these purposes